app/src/main/java/com/framgia/soundclound/service/MusicService.java

```diff
@@ -0,0 +1,324 @@
+            player.seekTo(position);
+        }
+    }
+
+    @Override
+    public void next() {
+        curentTrack = SharePreferences.getInstance().getIndex();
+        curentTrack++;
+        if (curentTrack > (mTracks.size() - 1)) {
+            curentTrack = 0;
+            SharePreferences.getInstance().putIndex(curentTrack);
+            play();
+        } else {
+            SharePreferences.getInstance().putIndex(curentTrack);
+            play();
+        }
+
+    }
+
+    @Override
+    public void prev() {
+        curentTrack = SharePreferences.getInstance().getIndex();
+        curentTrack--;
+        if (curentTrack < 0) {
+            curentTrack = mTracks.size() - 1;
+            SharePreferences.getInstance().putIndex(curentTrack);
+            play();
+        } else {
+            SharePreferences.getInstance().putIndex(curentTrack);
+            play();
+        }
+        SharePreferences.getInstance().putIndex(curentTrack);
+    }
+
+    @Override
+    public void onDestroy() {
+        stopForeground(true);
+        stopSelf();
+        super.onDestroy();
+
+    }
+
+    public int getDuration() {
+        return player.getDuration();
+    }
+
+    public void setShuffle() {
+        if (mShuffle) {
+            mShuffle = false;
+        } else {
+            mShuffle = true;
+        }
+    }
+
+    public void setRepeat() {
+        if (mRepeat) {
+            mRepeat = false;
+        } else {
+            mRepeat = true;
+        }
+    }
+
+    // // TODO: 15/1/2018
+    public void showNotifi() {
+        //notification
+        Intent notIntent = new Intent(this, PlayTrackActivity.class);
+        notIntent.setAction(Constant.ACTION_MAIN);
+        notIntent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_SINGLE_TOP);
+        PendingIntent pendInt = PendingIntent.getActivity(getApplicationContext(),
+                0,
+                notIntent, 0);
+
+        Intent previousIntent = new Intent(getApplicationContext(), MusicService.class);
+        previousIntent.setAction(Constant.ACTION_PREV);
+        PendingIntent ppreviousIntent = PendingIntent.getService(getApplicationContext(),
+                0,
+                previousIntent, PendingIntent.FLAG_UPDATE_CURRENT);
+
+        Intent playIntent = new Intent(getApplicationContext(), MusicService.class);
+        playIntent.setAction(Constant.ACTION_PLAY);
+        PendingIntent pplayIntent = PendingIntent.getService(getApplicationContext(),
+                0,
+                playIntent, PendingIntent.FLAG_UPDATE_CURRENT);
+
+        Intent nextIntent = new Intent(getApplicationContext(), MusicService.class);
+        nextIntent.setAction(Constant.ACTION_NEXT);
+        PendingIntent pnextIntent = PendingIntent.getService(getApplicationContext(),
+                0,
+                nextIntent, PendingIntent.FLAG_UPDATE_CURRENT);
+
+        Intent dismisIntent = new Intent(getApplicationContext(), MusicService.class);
```

Comment:
ERROR: [STYLE][DLS_DEAD_LOCAL_STORE] Dead store to dismisIntent in com.framgia.soundclound.service.MusicService.showNotifi()
http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE